cleanlab_studio/studio/studio.py

```diff
@@ -30,6 +31,8 @@
 if _pyspark_exists:
     import pyspark.sql
 
+warnings.simplefilter("always", DeprecationWarning)
```

Comment:
question: is there a reason to set this here? we generally wouldn't want to override a users warning filter configureation

Reply:
DeprecationWarning is filtered out by default usually. I can switch the warnings to the default UserWarning instead and remove this override. (see [docs](https://docs.python.org/3/library/warnings.html#warning-categories))

Reply:
https://docs.python.org/3/library/warnings.html#warning-categories

Ah looks like we can use `FutureWarning`,  which is both not ignored by default and is a better fit for what's happening here (we're not deprecating schema overrides, we're changing their behavior)